sdk/communication/azure-communication-sms/src/main/java/com/azure/communication/sms/SmsAsyncClient.java

```diff
@@ -38,117 +38,135 @@
 
     /**
      * Sends an SMS message from a phone number that belongs to the authenticated account.
+     * Phone number has to be in the format 000 - 00 - 00
      *
      * @param from Number that is sending the message.
-     * @param to The recipient's phone number. In this version, only one recipient in the list is supported.
+     * @param to The recipient's phone number.
      * @param message message to send to recipient.
-     * @param smsOptions set options on the SMS request, like enable delivery report, which sends a report
-     * for this message to the Azure Resource Event Grid.
-     * @param context the context of the request. Can also be null or Context.NONE.
      * @return response for a successful send Sms request.
      */
-    Mono<Response<SendSmsResponse>> sendMessageWithResponse(PhoneNumberIdentifier from,
-        List<PhoneNumberIdentifier> to, String message,
-        SendSmsOptions smsOptions, Context context) {
-        if (from == null) {
-            return monoError(logger, new NullPointerException("Argument 'from' cannot be null."));
-        } else if (to == null) {
-            return monoError(logger, new NullPointerException("Argument 'to' cannot be null."));
-        }
-
-        SendMessageRequest sendMessageRequest = createSmsMessageRequest(from, to, message, smsOptions);
-
-        return this.smsServiceClient.getSms().sendWithResponseAsync(sendMessageRequest, context);
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<SmsSendResult> send(String from, String to, String message) {
+        return send(from, to, message, null);
     }
 
     /**
      * Sends an SMS message from a phone number that belongs to the authenticated account.
      *
      * @param from Number that is sending the message.
-     * @param to The recipient's phone number. In this version, only one recipient in the
-     * list is supported.
+     * @param to The recipient's phone number.
      * @param message message to send to recipient.
+     * @param smsOptions set options on the SMS request, like enable delivery report, which sends a report
+     *                   for this message to the Azure Resource Event Grid.
      * @return response for a successful send Sms request.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<SendSmsResponse> sendMessage(PhoneNumberIdentifier from,
-        List<PhoneNumberIdentifier> to, String message) {
-        if (from == null) {
-            return monoError(logger, new NullPointerException("Argument 'from' cannot be null."));
-        } else if (to == null) {
-            return monoError(logger, new NullPointerException("Argument 'to' cannot be null."));
+    public Mono<SmsSendResult> send(String from, String to, String message,
+                                    SmsSendOptions smsOptions) {
```

Comment:
wondering if this is the right formatting for the cases when arguments are pushed to next line

Reply:
Agree that formatting looks wrong. The options argument can stay on the same line.